doc-content/enterprise-only/installation/clustering-bc-configure-standalone_proc.adoc

```diff
@@ -0,0 +1,56 @@
+[id='clustering-bc-configure-standalone_proc']
+= Configuring and running {CENTRAL} in a cluster
+Configure the {CENTRAL} cluster with Elasticsearch and the AMQ Broker, then start the cluster. 
+
+[NOTE] 
+====
+These steps describe a basic cluster configuration. For more complex configurations, see the https://access.redhat.com/documentation/en-us/red_hat_jboss_enterprise_application_platform/7.2.beta/html-single/configuration_guide/[_Red Hat JBoss EAP 7.2 Configuration Guide_].
+====
+
+.Prerequisites
+
+* Elasticsearch is installed as described in <<clustering-elasticsearch-proc_{context}>>.
+* AMQ Broker is installed and configured, as described in <<JMS-broker-configure_{context}>>.
+* {EAP} and {PRODUCT} are installed on each node of the cluster as described in <<clustering-download-eap-bc-proc>>.
+* An NFS server with a mounted partition is available.
+
+.Procedure
+. Open the `_EAP_HOME_/standalone/configuration/standalone-full-ha.xml` file in a text editor.
+. Edit or add the following properties to the `<system-properties>` element and replace the following placeholders:
+* `<AMQ_USER>` and `<AMQ_PASSWORD>` are the credentials that you defined when creating the AMQ Broker.
+* `<AMQ_BROKER_IP_ADDRESS>` is the IP address of the AMQ Broker.
+* `<ELASTICSEARCH_NODE_IP>` is the IP address where Elasticseach is installed.
+* `<SHARED_NETWORK_FOLDER>` is the network folder that all {CENTRAL}s in the cluster use to exchange data. This folder must be the same on all nodes of the cluster.
+
++
+[source,xml]
+----
+<system-properties>
+  <property name="appformer-jms-connection-mode" value="REMOTE"/>
+  <property name="appformer-jms-username" value="<AMQ_USER>  "/>
+  <property name="appformer-jms-password" value="<AMQ_USER_PASSWORD>"/>
+  <property name="appformer-jms-url" value="tcp://<AMQ_BROKER_IP_ADDRESS>:61616?ha=true&retryInterval=1000&retryIntervalMultiplier=1.0&reconnectAttempts=-1"/>
+  <property name="org.appformer.ext.metadata.elastic.port" value="9300"/>
+  <property name="org.appformer.ext.metadata.elastic.host" 
+      value="<ELASTICSEARCH_NODE_IP>"/>
+  <property name="org.appformer.ext.metadata.elastic.cluster" 
+      value="kie-cluster"/>
+  <property name="org.appformer.ext.metadata.index" value="elastic"/> 
+  <property name="org.appformer.ext.metadata.elastic.retries" value="10"/>
+  <property name="org.uberfire.nio.git.dir" value="<SHARED_NETWORK_FOLDER>"/>
+</system-properties>
+----
+. Save the `standalone-full-ha.xml` file.
+. To start the cluster, navigate to `__EAP_HOME__/bin` and enter one of the following commands:
+** On Linux or UNIX-based systems:
++
+[source,bash]
+----
+$ ./standalone.sh -c standalone-full-ha.xml
+----
+** On Windows:
++
+[source,bash]
+----
+standalone.bat -c standalone-full-ha.xml
+----
```

Comment:
Not here, but in "2.5 Verifying the [DM/PAM] cluster", "The Assets window" should be "The *Assets* window". Also, not sure what your opinion is on example URLs being rendered as active/clickable (blue) in the output, but to disable them, add a \ in front: `\http:://localhost...`  The forward slash isn't rendered in the output and disables the link so that it's just text.